core/frontend/src/main/scala/io/udash/bindings/modifiers/DOMManipulator.scala

```diff
@@ -0,0 +1,41 @@
+package io.udash.bindings.modifiers
+
+import org.scalajs.dom.{Element, Node}
+
+private[bindings]
+trait DOMManipulator {
+  import DOMManipulator._
+
+  /**
+    * Provides custom child elements replace method. This method takes
+    * root element, old children and new children. It should return `true`,
+    * if it does not replace elements in DOM.
```

Comment:
The result requirements should be explained, so the user understands what that boolean triggers.